dune/gdt/local/numerical-fluxes/lax-friedrichs.hh

```diff
@@ -54,23 +60,44 @@ public:
 
   using BaseType::apply;
 
-  StateRangeType apply(const StateRangeType& u,
-                       const StateRangeType& v,
-                       const PhysicalDomainType& n,
-                       const XT::Common::Parameter& param = {}) const override final
+  StateType apply(const I& intersection,
+                  const LocalIntersectionCoords& x,
+                  const StateType& u,
+                  const StateType& v,
+                  const PhysicalDomainType& n,
+                  const XT::Common::Parameter& param = {}) const override final
   {
-    const auto lambda =
-        1. / std::max(this->flux().jacobian(u, param).infinity_norm(), this->flux().jacobian(v, param).infinity_norm());
-    return 0.5 * ((this->flux().evaluate(u, param) + this->flux().evaluate(v, param)) * n) + 0.5 * ((u - v) / lambda);
+    const auto local_flux = this->flux().local_function();
```

Comment:
I would think that you would need an
```c++
auto local_flux_inside = this->flux().local_function();
local_flux_inside->bind(intersection.inside());
auto local_flux_outside = this->flux().local_function();
local_flux_outside->bind(intersection.neighbor() ? intersection.outside() : intersection.inside());
```

Reply:
Oh, you are right. We probably should have tests actually using x-dependent fluxes at some point :).